projects/1.19/assets/create-gourmet/gourmet/lang/zh_cn.json

```
@@ -0,0 +1,79 @@
+{
+  "item.gourmet.raw_bacon": "生培根",
+  "item.gourmet.cooked_bacon": "熟培根",
+  "item.gourmet.caramelized_bacon": "焦糖培根",
+  "item.gourmet.raw_drumstick": "生鸡腿",
+  "item.gourmet.cooked_drumstick": "熟鸡腿",
+  "item.gourmet.fried_chicken": "炸鸡",
+  "item.gourmet.butter": "黄油",
+  "item.gourmet.cheese": "奶酪",
+  "item.gourmet.croissant": "牛角包",
+  "item.gourmet.croissant_dough": "牛角包面团",
+  "item.gourmet.incomplete_croissant_dough": "牛角包面团片",
+  "item.gourmet.sweet_berry_danish": "甜浆果丹麦酥",
+  "item.gourmet.glow_berry_danish": "发光浆果丹麦酥",
+  "item.gourmet.cheese_danish": "奶酪丹麦酥",
+  "item.gourmet.sweet_berry_danish_dough": "甜浆果丹麦酥面团",
+  "item.gourmet.glow_berry_danish_dough": "发光浆果丹麦酥面团",
+  "item.gourmet.cheese_danish_dough": "奶酪丹麦酥面团",
+
+  "item.gourmet.dumpling_wrapper": "包子皮",
+  "item.gourmet.incomplete_dumpling": "包子（半成品）",
+  "item.gourmet.dumpling": "包子",
+
+  "item.gourmet.ice_cream_scoop": "冰淇淋球",
+  "item.gourmet.chocolate_ice_cream_scoop": "巧克力冰淇淋球",
+  "item.gourmet.sweet_berry_ice_cream_scoop": "甜浆果冰淇淋球",
+  "item.gourmet.ice_cream_cone": "冰淇淋球甜筒",
+  "item.gourmet.chocolate_ice_cream_cone": "巧克力冰淇淋甜筒",
+  "item.gourmet.sweet_berry_ice_cream_cone": "甜浆果冰淇淋甜筒",
+  "item.gourmet.neapolitan_ice_cream_cone": "那不勒斯冰淇淋甜筒",
+
+  "item.gourmet.carrot_cake": "胡萝卜蛋糕",
+  "item.gourmet.carrot_cake_base": "胡萝卜蛋糕基底",
+  "item.gourmet.incomplete_carrot_cake": "胡萝卜蛋糕（半成品）",
+
+  "item.gourmet.variety_dessert_platter": "缤纷甜品拼盘",
+  "item.gourmet.comforting_dumpling_meal": "温馨包子餐",
+  "item.gourmet.southern_breakfast": "南式早餐",
+
+  "item.gourmet.flatcake": "美式松饼",
+  "item.gourmet.incomplete_flatcake_stack": "美式松饼塔（半成品）",
+  "item.gourmet.flatcake_stack": "美式松饼塔",
+  "item.gourmet.flatcake_batter": "美式松饼糊",
+  "item.gourmet.waffle": "华夫饼",
+  "item.gourmet.waffle_batter": "华夫饼糊",
+  "item.gourmet.waffle_stack": "华夫饼塔",
+  "item.gourmet.incomplete_waffle_stack": "华夫饼塔（半成品）",
+  "item.gourmet.french_toast": "法国吐司",
+  "item.gourmet.fried_egg": "煎蛋",
+
+  "item.gourmet.incomplete_burger": "汉堡（半成品）",
+  "item.gourmet.bacon_burger": "培根汉堡",
+  "item.gourmet.big_bacon_burger_meal": "大号培根汉堡套餐",
+  "item.gourmet.beef_patty": "牛肉饼",
+  "item.gourmet.ground_beef": "绞牛肉",
+  "item.gourmet.buns": "小圆面包",
+  "item.gourmet.ice_cube": "冰块",
+  "item.gourmet.carrot_slices": "胡萝卜片",
+
+  "fluid_type.gourmet.seed_oil": "籽油",

```

```suggestion
  "fluid_type.gourmet.seed_oil": "种子油",

```

```suggestion
  "fluid_type.gourmet.seed_oil": "植物油",

```

籽油还真没听过，叫种子油的多（GTCE），植物油的也有点